src/System.IO.Pipelines/System/IO/Pipelines/FlushResult.cs

```diff
@@ -23,12 +30,12 @@ public FlushResult(bool isCancelled, bool isCompleted)
         }
 
         /// <summary>
-        /// True if the currrent flush was cancelled
+        /// True if the current <see cref="PipeWriter.FlushAsync"/> operation was canceled otherwise false
```

Comment:
nit: Change to one of (here and elsewhere):
`True if the current <see cref="PipeWriter.FlushAsync"/> operation was canceled, otherwise false.`
`True if the current <see cref="PipeWriter.FlushAsync"/> operation was canceled. Otherwise false.`